dotnet/src/Connectors/Connectors.OpenAI.UnitTests/Core/OpenAIFunctionToolCallTests.cs

```diff
@@ -82,4 +82,27 @@ public void ConvertToolCallUpdatesWithNotEmptyIndexesReturnsNotEmptyToolCalls()
         Assert.Equal("test-function", toolCall.FunctionName);
         Assert.Equal("test-argument", toolCall.FunctionArguments.ToString());
     }
+
+    [Fact]
+    public void TrackStreamingToolingUpdateWithNullUpdatesDoesNotThrowException()
```

Comment:
This doesn't cover the new code path. Can you run the test with code coverage to verify the new code path is being tested.